src/beamlime/applications/mixins.py

```diff
@@ -0,0 +1,180 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+import asyncio
+from logging import DEBUG, ERROR, INFO, WARN
+
+from ..core.schedulers import async_timeout
+from ..logging.loggers import BeamlimeLogger
+
+MAX_INSTANCE_PAUSED = 20
+
+
+class LogMixin:
+    """
+    Logging interfaces
+
+    Protocol
+    --------
+    BeamlimeLoggingProtocol
+    """
+
+    def _log(self, level: int, msg: str, args: tuple):
+        if isinstance(self.logger, BeamlimeLogger):
+            self.logger._log(level=level, msg=msg, args=args, app_name=self.app_name)
```

Comment:
Yes indeed... I intentionally forgot to add properties in the `Protocol`
since it doesn't allow `issubclass` when it has non-method members... I will add them.